src/js/views/patients/patient/flow/patient-flow.scss

```diff
@@ -17,9 +17,8 @@
 .patient-flow__sidebar {
   display: flex;
   flex: initial;
-  max-width: 400px;
   min-height: 0;
-  min-width: 256px;
+  width: 256px;
 }
```

Comment:
This currently expands up to width `400px` if the patient's name is long. This sets it to not expand, and stay at `256px` no matter what.

We did the same thing for the patient dashboard sidebar in a previous pull request: https://github.com/RoundingWell/care-ops-frontend/pull/715/files#diff-12b61ecbc123f2512003e721788f704753df710a9822a5ef7d34ebd21c5ce2c6R21